frontend/src/app/components/search-form/search-form.component.html

```diff
@@ -1,7 +1,7 @@
 <form [formGroup]="searchForm" (submit)="searchForm.valid && search()" novalidate>
   <div class="d-flex">
     <div class="search-box-container mr-2">
-      <input autofocus (focus)="focus$.next($any($event).target.value)" (click)="click$.next($any($event).target.value)" formControlName="searchText" type="text" class="form-control" i18n-placeholder="search-form.searchbar-placeholder" placeholder="Explore the full Bitcoin ecosystem">
+      <input [autofocus]="searchAutofocus" (focus)="focus$.next($any($event).target.value)" (click)="click$.next($any($event).target.value)" formControlName="searchText" type="text" class="form-control" i18n-placeholder="search-form.searchbar-placeholder" placeholder="Explore the full Bitcoin ecosystem">
```

Comment:
I believe you can replace the whole custom component with a conditional attribute like:

`[attr.autofocus]="boolean"`

Reply:
Does not work. But I replaced with a `.focus()` call in the component instead of using a directive, less code.